fixtures/events/performance_problems/sql-injection/sql-injection-test-comment.json

```diff
@@ -0,0 +1,158 @@
+{
```

Comment:
should this correspond to a test in `test_sql_injection_detector.py`, similar to e.g. `test_sql_injection_not_in_where`?

Reply:
good catch! that commit didn't make it through, will add